.travis.yml

```diff
@@ -65,3 +65,14 @@ script:
   - if [ -n "$CLANG_FORMAT_PATH" ]; then bin/clangfmt --test; fi
   - bin/scalafmt --test
   - bash sbt -J-Xmx2G rebuild "set scriptedBufferLog in sbtScalaNative := false" test-all
+
+before_cache:
+  - find $HOME/.sbt -name "*.lock" | xargs rm
+  - find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm
+
+cache:
+  apt: true
```

Comment:
This doesn't seem to be sufficient, it still downloads things from LLVM repo:

https://travis-ci.org/scala-native/scala-native/jobs/239922054#L242